### PR TITLE
Add an indicator for the  eBPF sourced entries

### PR DESCRIPTION
**Pull request by mertyildiran**
### eBPF mascot bee (current)

Screenshot from 2022-03-12 03-15-17

Screenshot from 2022-03-12 03-18-26

### Padlock (alternative)

Screenshot from 2022-03-12 03-23-42

Screenshot from 2022-03-12 03-23-57

### CODE REVIEW

**Review by gadotroee — COMMENTED** [review]
I checked all backend side and it looks good. I want someone from frontend to look on the change in there but anyway we will probably wait with it after a big refactor that we currently finishing of splitting some if the UI code to be shared between mizu and mizu enterprise

File: tap/api/api.go [hunk 1]
```diff
@@ -44,6 +44,15 @@ type Extension struct {
...
 }
 
+type Capture string
+
+const (
+	Pcap    Capture = "pcap"
+	Envoy   Capture = "envoy"
+	Linkerd Capture = "linkerd"
+	Ebpf    Capture = "ebpf"
+)
```
Comment (nimrod-up9) [inline tap/api/api.go:54]: Add `UndefinedCapture Capture = ""` in order to handle uninitialized `Capture` variables.

Comment (mertyildiran) [reply tap/api/api.go]: @nimrod-up9 done with https://github.com/up9inc/mizu/commit/2cbc54036a47c333012092c3146d46d92b87dcf2

**Comment by gadotroee** [comment]
@mertyildiran , PR https://github.com/up9inc/mizu/pull/883 was merged yesterday so now this can be merged but because it was included big change in the UI there are small conflicts here.

I think you can solve them and I will take a look to approve and insert this feature

**Comment by mertyildiran** [comment]
@gadotroee conflicts are resolved.

File: ui-common/src/components/TrafficViewer/EntryListItem/EntryListItem.tsx [hunk 2]
```diff
@@ -154,6 +156,17 @@ export const EntryItem: React.FC<EntryProps> = ({entry, style, headingMode}) =>
                 protocol={entry.proto}
                 horizontal={false}
             /> : null}
+            {/* TODO: Update the code below once we have api.Pcap, api.Envoy and api.Linkerd distinction in the backend */}
+            {entry.capture === "ebpf" ? <div className={styles.capture}>
```
Comment (gadotroee) [inline ui-common/src/components/TrafficViewer/EntryListItem/EntryListItem.tsx:160]: extract to some const. (the magic word "ebpf")

**Review by gadotroee — APPROVED** [review]
I don't want to wait to much with this change. The change looks ok IMO, Im approving it but just know that currently our acceptance tests are failing (due to the change in the common split) so if you merge we will see failure that expected. And we are working on fix for the tests

**Pull request merged**